### ISSUE TITLE
[next] URLs to internal pages require trailing slash 

### ISSUE THREAD

**Issue by andrico1234** [modernweb-dev/rocket#336]
In the Rocket site the following link works perfectly fine:

```
Once you have a project running continue with [the next steps](/docs/setup/adding-pages/).
```

Removing the trailing slash causes a 404 for a number of the static site. Note: this only happens when running the site locally

The following is a screenshot of the failing network requests when I remove the trailing slash. 
<img width="699" alt="image" src="https://user-images.githubusercontent.com/17087167/159867748-e00dd8dd-6c45-4dd1-b1e8-67c7cbf89791.png">

It seems that when a trailing slash is present, the url to retrieve one of the static files is:

`http://localhost:8000/__wds-outside-root__/1/packages/drawer/rocket-drawer.js`

while it's 

`http://localhost:8000/packages/drawer/rocket-drawer.js`

when the trailing slash doesn't exist.

**Comment by daKmoR** [comment]
yes the trailing slash is mandatory... imho it's the safest option

https://next.rocket.modern-web.dev/docs/basics/routing/

**Comment by andrico1234** [comment]
Thanks! I don't know how I missed this page, I had a quick look through the docs before I posted this 🤦🏽‍♂️
